django/core/mail/message.py

```diff
@@ -100,22 +100,62 @@ def forbid_multi_line_headers(name, val, encoding):
     return str(name), val
 
 
+def split_addr(addr, encoding):
+    """
+    Splits an address into local part and domain, properly encoded.
```

Comment:
Use PEP257 verb style for new docstrings ("Split...."). Describe what "properly encoded" means?